_posts/2020-11-31-homebrew-2.6.0.md

```diff
@@ -0,0 +1,36 @@
+---
+title: 2.6.0
+author: MikeMcQuaid
+---
+Today I'd like to announce Homebrew 2.6.0. The most significant changes since 2.5.0 are macOS Big Sur support, `brew` commands replacing all `brew cask` commands, the beginnings of macOS M1/Apple Silicon/ARM support and API deprecations.
+
+Major changes and deprecations since 2.5.0:
+
+- [macOS Big Sur is supported (and High Sierra unsupported)](https://github.com/Homebrew/brew/pull/9054)
+- [`depends_on :java`, `brew switch`, `brew diy` and various other APIs have been deprecated](https://github.com/Homebrew/brew/pull/9209)
+- [All `brew cask` commands have been deprecated in favour of `brew` commands (with `--cask`) when necessary](https://github.com/Homebrew/brew/pull/8899)
+- [`Homebrew.args` is deprecated](https://github.com/Homebrew/brew/pull/8795)
+- [All `Requirement`s are deprecated in Homebrew/core](https://github.com/Homebrew/brew/pull/9052)
+- [macOS Homebrew on M1/Apple Silicon/ARM has partial functionality. We recommend installing into `/opt/homebrew` and forbid installing into `/usr/local` (to avoid clashing with the macOS Intel install and allow their usage side-by-side).](https://github.com/Homebrew/brew/pull/9117)
+- [`brew tap-new` will setup GitHub Actions workflows to](https://github.com/Homebrew/brew/pull/8762) [upload to GitHub Releases](https://github.com/Homebrew/brew/pull/8410). Read [the blog post](https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/) for more documentation.
+- [GitHub deprecated their API's basic authentication](https://github.com/Homebrew/brew/pull/8826)
+
+Other changes since 2.5.0 I'd like to highlight are the following:
+
+- [`brew doctor` checks all tap branches, not just `homebrew-core`](https://github.com/Homebrew/brew/pull/9305)
```

Comment:
The wording's slightly hard to parse without looking it up - I had to look at the PR to interpret what this one meant. How about this wording?
```suggestion
- [`brew doctor` checks the active branch for all taps, not just Homebrew/homebrew-core](https://github.com/Homebrew/brew/pull/9305)
```